packages/packages/icons/package.json

```diff
@@ -5,5 +5,8 @@
   },
   "peerDependencies": {
     "react": "17.x"
+  },
+  "elementor": {
+    "type": "util"
```

Comment:
Fix indention in all `package.json` files

Reply:
Done.